### ISSUE TITLE
T12 dynamic tags trigger missing tags rule

### ISSUE THREAD

**Issue by MattToczek** [terraform-linters/tflint#949]
When dynamic tag blocks are used like so:

```
  dynamic "tag" {	 
    for_each = var.common_tags	
    content {	
      key                 = tag.key	
      value               = tag.value	
      propagate_at_launch = true	
    }	
  }
```

The AWS rule aws_resource_missing_tags is triggered, despite the tags being assigned the resource. 

it can be seen in `terraform/modules/ecs-user/asg.tf` in this pr https://github.com/dwp/orchestration-service/pull/175
